<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>代替密码破译v1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>841</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_VedioShow">
     <attribute name="title">
      <string>代替密码破译界面</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>明文</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>密文</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCracked">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>破解</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonEncryption">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>370</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>加密</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditCiphertext">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>400</width>
        <height>330</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditCleartext">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>400</width>
        <height>330</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBoxKey">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>391</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>原始的密匙：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditSetPwd">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>LFWOAYUISVKMNXPBDCRJTQEGHZ</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>破解的密匙：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditGeneratePwd">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButtonGeneratePwd">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>430</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>产生随机密匙</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonClearHackMessage">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>370</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>清空密文</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonClearMessage">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>370</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>清空明文</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>其他</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>844</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
